docs/team_members.rst

```diff
@@ -0,0 +1,38 @@
+============
+Team Members
+============
+
+## Research Team 
+
+The core Web3 Foundation research team working on various areas relevant to decentralised systems.
+
+.. toctree::
+   :glob:
+   :maxdepth: 1
+
+   team_members_research/alfonso
```

Comment:
Would it be more future-proof to just have everyone be listed under the same directory `team_members/$name`, and only have the structure be expressed on this page? Syed is also in both teams, so that would be less confusing that way.

Reply:
As I did not add the glob instruction together with the `team_members_research/*` entry, I did not feel like I should change that. But I agree that the mix of globbing and hard-coded is rather ugly. I see two options:

- No hard coding (apart from Syed), two separate folders resulting in an alphabetic list. Might require all file names being lower case.
- Only hard-coded paths, just one folder as suggested.

Reply:
Oh right yes I forgot about the glob, but I think it's OK to drop that too. I think sphinx will warn if there are any orphan pages underneath team_members/ that are not referenced by any other pages.